TWCManager.py

```diff
@@ -97,6 +97,15 @@
     debugLog(1, "Unable to find a configuration file.")
     sys.exit()
 
+
+
+
+########################################################################
+# Write the PID in order to let a supervisor restart it in case of crash
+PIDTWCManager=open("/home/pi/TWCManager/TWCManager.pid","w")
+PIDTWCManager.write(str(os.getpid()))
+PIDTWCManager.close()
+
```

Comment:
This hardcoded path would fail on any installation where the directory specified doesn't exist

Reply:
you are right, it should be change for:
```
PIDfile=config["config"]["settingsPath"] + "/TWCManager.pid"
PIDTWCManager=open(PIDfile,"w")
PIDTWCManager.write(str(os.getpid()))
PIDTWCManager.close()
```
May I submit a new PR?

Reply:
You can push a new commit to your branch, and that will then update the changes in the PR :)